### ISSUE TITLE
[DOC] Public docs to flagd.dev

### ISSUE THREAD

**Issue by beeme1mr** [open-feature/flagd#763]
### Change in the documentation

- [x] Identify the doc tooling that should be used
- [x] Publish docs to Netlify after a successful merge to main
- [ ] Create issues to expand on areas in the docs that need more information.
- [ ] Covert existing docs to work with the tool
- [ ] Update existing references to the GitHub markdown docs to the website

**Comment by agardnerIT** [comment]
I feel quite strongly that the GitHub `.md` and a "proper website" should be separated. They serve different purposes for different audiences. That is not to say that a large portion of the content cannot be re-used. Indeed on my skeleton site (link later) you'll notice large chunks are copy / paste from the docs already here.

The Github readme's will remain closest to the code and the primary audience will be developers and those "very close" to the code. The secondary purpose of these files should be as signposts out to the official documentation site.

The "proper website" will be higher level, more accessible, written in more plain English and (perhaps crucially for adoption) easily indexable by search engines and LLMs.

We should not (IMO) just re-use what we already have in this repo as a documentation site.

While I appreciate this will be extra work, the additional overhead will be worth it for user adoption.

I am happy to take point on this responsibility.

As part of this initiative, I have a sample website (skinned with readthedocs theme) which I'm ready to copy / paste the `.md` files over to here: https://agardnerit.github.io/flagddocs/ (repo: https://github.com/agardnerit/flagddocs).

Once the `.md` files are here, I'll remove that temporary site + repo.

**Comment by agardnerIT** [comment]
For other readers, Mike and I had a chat as there was a misunderstanding. Indeed I fully support the docs in this repo.

My point was that READMEs in the repo aren't necessarily the right place (or content) for a "proper website). We both agree on this.

TLDR: https://flagd.dev is live now and is built entirely on the content from the /web-docs folder in this repo. The original (more technical) docs remain in `/docs`.

I think this is probably ready to be closed.
